content/chapters/compute/lab/content/processes.md

```diff
@@ -250,11 +250,11 @@ Notice what each of the two processes prints:
 Unlike `system()`, who also waits for its child, when using `fork()` we must do the waiting ourselves.
 In order to wait for a process to end, we use the [`waitpid()`](https://linux.die.net/man/2/waitpid) syscall.
 It places the exit code of the child process in the `status` parameter.
-This argument is actually a bitfield containing more information than merely the exit code.
+This argument is actually a bit field containing more information than merely the exit code.
```

Comment:
```suggestion
This argument is actually a bit-field containing more information than merely the exit code.
```
I've seen it written with a dash more than separately: https://en.cppreference.com/w/cpp/language/bit_field.